### PR TITLE
Bug#1493968 - added example for label-scoped multiproject quotas

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1493968

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@jrosenta PTAL

**Comment by gaurav-nelson** [comment]
Thanks @jrosenta for reviewing this (https://bugzilla.redhat.com/show_bug.cgi?id=1493968#c4)

**Comment by gaurav-nelson** [comment]
@openshift/team-documentation PTAL 🌗

File: admin_guide/multiproject_quota.adoc [hunk 1]
```diff
@@ -22,21 +22,20 @@ projects.
 [[multi-project-quotas-selecting-projects]]
 == Selecting Projects
 
-Projects can be selected based on either annotation selection, label selection, or both.
-For example:
+Projects can be selected based on annotation selection, label selection, or
```
Comment (kalexand-rh) [inline admin_guide/multiproject_quota.adoc]: "Projects can be selected" should be "You can select projects"

File: admin_guide/multiproject_quota.adoc [hunk 2]
```diff
@@ -75,12 +74,45 @@ status:
 <3> A label selector that can be used to select projects.
 <4> A per-namespace map that describes current quota usage in each selected project.
 <5> The aggregate usage across all selected projects.
-====
 
 This multi-project quota document controls all projects requested by
 *<user-name>* using the default project request endpoint. You are limited to 10
 pods and 20 secrets.
 
+Similarly, to select projects based on labels, run the command:
```
Comment (kalexand-rh) [inline admin_guide/multiproject_quota.adoc]: I'd say "this command" instead of "the command"

File: admin_guide/multiproject_quota.adoc [hunk 3]
```diff
@@ -75,12 +74,45 @@ status:
...
+[source, bash]
+----
+$  oc create clusterresourcequota for-name \ <1>
+    --project-label-selector=name=frontend \ <2>
+    --hard=pods=10 --hard=secrets=20
+----
+
+<1> Both `clusterresourcequota` and `clusterquota` are aliases of the same
+command. `for-name` is the name of the `clusterresourcequota` object.
+<2> A label selector to select projects in the format
```
Comment (kalexand-rh) [inline admin_guide/multiproject_quota.adoc]: Maybe "To select projects by label, provide a key-value pair using the format `--project-label-selector=key=value`."

**Comment by gaurav-nelson** [comment]
Thanks @kalexand-rh I have applied changes. Can you please mark this as reviewed if there are no more changes?

**Comment by kalexand-rh** [comment]
@gaurav-nelson, thanks! My apologies for missing the label swap.

**Comment by xiuwang** [comment]
LGTM

**Pull request merged**